comps/reranks/README.md

```diff
@@ -0,0 +1,39 @@
+# ReRank Microservice
+
+## Introduction
+
+Provides functionality for reranking outputs.
+It allows for fine-tuning the results based on specific criteria, improving the overall quality and relevance for the user.
+
+## Key Features
+
+- **Configurable Ranking Criteria**: Define custom ranking rules based on various factors like relevance, creativity, style, or user-defined metrics.
```

Comment:
Sounds like a cool feature but I don't see pointers to where and how I would configure.